signer/src/wsts_state_machine.rs

```diff
@@ -45,7 +45,23 @@ pub enum StateMachineId {
     /// Identifier for a Bitcoin signing state machines
     BitcoinSign(SigHash),
     /// Identifier for a rotate key verification signing round
-    RotateKey(PublicKeyXOnly, model::BitcoinBlockHash),
+    DkgVerification(PublicKeyXOnly, model::BitcoinBlockRef),
+}
+
+impl std::fmt::Display for StateMachineId {
```

Comment:
Could probably have used `strum` for this, but I wasn't sure off the bat how to handle the formatting of the inner's, but if anyone else knows we could change it.

Reply:
You can, see https://docs.rs/strum_macros/latest/strum_macros/derive.Display.html, but what is here is good too. So I'm okay keeping it as is.

Reply:
Someone can boyscout it when they're passing by this code the next time :)

Reply:
Yeah, same for `dkg::verification::State`